connectors/byoc.py

```diff
@@ -656,6 +637,29 @@ async def prepare(self, config):
                 f"Could not update service type/configuration for connector {self.id}"
             )
 
+    async def validate_filtering(self, validator):
+        draft_filter = self.filtering.get_draft_filter()
+        if not draft_filter.has_validation_state(FilteringValidationState.EDITED):
```

Comment:
In Ruby we've a debug log , when we skip validation. Should we add it here, too? _Could_ make it easier to debug.

`logger.debug(f"Filtering of connector {self.id} is in state {draft_filter["validation"]["state"]}, skipping validation.")`